hathor/cli/util.py

```diff
@@ -212,18 +213,30 @@ def kwargs_formatter(_, __, event_dict):
             event_dict['event'] = event_dict['event'].format(**event_dict)
         return event_dict
 
+    processors: List[Any] = [
+        structlog.stdlib.filter_by_level,
+        structlog.stdlib.add_logger_name,
+        structlog.stdlib.add_log_level,
+    ]
+
+    try:
+        from structlog_sentry import SentryProcessor
+    except ModuleNotFoundError:
+        pass
+    else:
+        processors.append(SentryProcessor(level=logging.ERROR))
```

Comment:
I was going to suggest that we add a `enable_sentry` param and gate the whole `try` block on it. But I understand how this can get complicated. I have a branch where I refactor the argparse structure that would make this possible, I'll add this to a TODO. I think it would really be better to not try to load this at all when we don't need it, and error when we need it but fail to load it.

The way it currently is it might fail silently if the user only installs `sentry_sdk` but not `structlog_sentry`.

Reply:
You're right. I fixed the silent failure now. Can you review it again, please?

Reply:
Makes sense, I think it's good now.